test/zdtm.py

```diff
@@ -618,11 +622,15 @@ def available():
                 ["make", "zdtm_ct"], env=dict(os.environ, MAKEFLAGS=""))
         if not os.access("zdtm/lib/libzdtmtst.a", os.F_OK):
             subprocess.check_call(["make", "-C", "zdtm/"])
+        if opts['rootless'] or os.getuid() != 0:
```

Comment:
we don't need to check uid. rootless means that we want to test the unprivileged mode.

Reply:
Fixed in multiple places, thanks.